scripts/run_performance_job.py

```diff
@@ -474,7 +474,7 @@ def run_performance_job(args: RunPerformanceJobArgs):
     configurations = { "CompilationMode": "Tiered", "RunKind": args.run_kind }
 
     using_mono = False
-    if mono_dotnet is not None:
+    if mono_dotnet is not None or mono_aot:
```

Comment:
It seems this will pass exclusions for NoMono and NoAOT for mono_aot runs, it seems that the previous setup would have instead had NoAOT and NoWASM (https://github.com/dotnet/runtime/blob/00cb272da591b6c5466a294260a5f063f2960fac/eng/testing/performance/performance-setup.sh#L356-L364). I am not sure if NoWASM was correct in the first place, but is something we should double check/watch for in the test run.